shynet/core/management/commands/sanity_checks.py

```diff
@@ -0,0 +1,49 @@
+import traceback
+import uuid
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.crypto import get_random_string
+
+from django.db import DEFAULT_DB_ALIAS, connections
+from django.db.utils import OperationalError, ConnectionHandler
+from django.core.exceptions import ImproperlyConfigured
+
+from core.models import User
+
+
+class Command(BaseCommand):
+    help = "Performs safe checks on the Shynet setup"
```

Comment:
Grammar fix; can we change to "Performs sanity checks on the Shynet setup"?

Reply:
done